cmd/admin.go

```diff
@@ -283,6 +283,9 @@ func runCreateUser(c *cli.Context) error {
 	if c.IsSet("password") {
 		password = c.String("password")
 	} else if c.IsSet("random-password") {
+		if err := initDB(); err != nil {
```

Comment:
initDB() should not be called twice. You should move the call at line 299 to before line 281.

Reply:
Sorry!
This is my mistake.